### DISCUSSION TITLE
What's the default relation sort order?

### DISCUSSION THREAD

**Discussion by jwld** [prisma/prisma#21637] [Q&A] [answered]
### Question

Relations have always appeared to be sorted by createdAt, with the latest one last in the list. But recently we're seeing relations coming back in the reverse order. This has quite serious implications for our codebase - I can't actually see anything in the docs about this. Should we always be adding an `orderBy` when we include relations in a query?

**Comment by ludralph** [comment, marked as answer]
Hi @jwld 👋 

Thank you for raising this question.

The documentation on [sorting](https://www.prisma.io/docs/concepts/components/prisma-client/filtering-and-sorting#sorting) states that you should use the [orderBy](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#orderby) option to sort a list of records or a nested list of records by a particular field or set of fields. It also says you should use the `orderBy` option when you include relations in a query to ensure the results are sorted in the order you expect. This is especially important if your codebase relies on the order of the returned relations.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by jwld** [reply]
That's good to know, thank you!

**Reply by SansPapyrus683** [reply]
Hi, I was curious what the default order is when the `orderBy` option is not provided.

**Reply by rcidaleassumpo** [reply]
@ludralph you did not answer the question. The question is about what's the default, not whether or not you should use the orderBy.

**Reply by brunobely** [reply]
I don't know this for sure, but I would be surprised if the default sort order wasn't the underlying DBMS's default sort order ([in many cases, there isn't one](https://www.reddit.com/r/PostgreSQL/comments/1k6nojy/what_is_the_default_order_in_postgresql_if_there/)).

**Comment by acidjazz** [comment]
It would be cool if we could specify a default sorting at the model level
